modules/ROOT/pages/6.0-release-notes-options.adoc

```diff
@@ -0,0 +1,76 @@
+= TinyMCE 6.0 Options changes
+:navtitle: TinyMCE 6.0 Options changes
+:description: TinyMCE 6.0 Options changes
+:keywords: releasenotes, options, configurations
+
+This section documents the changes made to {productname} 6.0 Options.
+
+It also documents the {productname} 6.0 Option that has been removed.
+
+* xref:options[Options]
+* xref:changed-options[Changed Options]
+* xref:removed-or-deprecated-apis[Removed or deprecated Options]
+
+// tag::options[]
+[[options]]
+== Options
+
+
+[[changed-options]]
+=== Changed options
+
+
+[[element-format]]
+==== `element_format`
+
+Previously, `element_format` had no explicit default value. It’s implicit value, however, was `xhtml`.
+
+In {productname} 6.0, `element_format` now has an explicit default value: `html`.
+
+
+[[image-upload-handler]]
+==== `image_upload_handler`
+
+Previously the `images_upload_handler` option passed `success` or `failure` callbacks to report the status of an image upload.
+
+In {productname} 6, `images_upload_handler` reports upload results by returning a `Promise`, with a value of either `resolve` or `reject`.
+
+
+[[link-default-protocol]]
+==== link_default_protocol
+
+The `link_default_protocol` option sets the link protocol used by links added or edited using the _link_ dialog.
+
+It now defaults to `https`. Previously it defaulted to `http`.
+
+NOTE: The `link_default_protocol` value is only applied to an edited or inserted link if the protocol is not explicitly specified.
+
+[[tinymce-active-editor-options-get]]
+==== tinymce.activeEditor.options.get
+
+Previously `tinymce.activeEditor.options.get('plugins')` returned the configured plugins as a string.
+
+As of {productname} 6, `tinymce.activeEditor.options.get('plugins')` now returns the configured plugins as an array.
+
+
+//=== Fixed options
+
+//=== Added options
+
+
+[[removed-or-deprecated-options]]
+=== Removed or deprecated options
+
+
+[[textpattern]]
+==== TextPattern
+
+* The `textpattern_patterns` option is no longer an option. It has been renamed to `text_patterns` and is now a Core part of {productname} 6.0.
++
+All references to `textpattern_patterns` should be removed from configurations.
```

Comment:
This doesn't seem correct? We want them instead to rename `textpattern_patterns` to `text_patterns`